### PR TITLE
dummy formatting errors

### PR DESCRIPTION
**Pull request by MarcelKoch**

### CODE REVIEW

**Comment by MarcelKoch** [comment]
check-format!

**Comment by MarcelKoch** [comment]
format!

File: .github/workflows/check-formatting.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Check formatting
+on:
+  workflow_call:
```
Comment (yhmtsai) [inline .github/workflows/check-formatting.yml:3]:
```suggestion
on: workflow_call
```

File: .github/format-rebase.sh [hunk 2]
```diff
@@ -30,7 +30,7 @@ git rebase --rebase-merges --empty=drop --no-keep-empty \
     --exec "cp /tmp/add_license.sh /tmp/format_header.sh /tmp/update_ginkgo_header.sh dev_tools/scripts/ && \
             dev_tools/scripts/add_license.sh && dev_tools/scripts/update_ginkgo_header.sh && \
             for f in \$($DIFF_COMMAND | grep -E '$FORMAT_HEADER_REGEX'); do dev_tools/scripts/format_header.sh \$f; done && \
-            for f in \$($DIFF_COMMAND | grep -E '$FORMAT_REGEX'); do $CLANG_FORMAT -i \$f; done && \
+            pipx run pre-commit run --files $TO_FORMAT && \
```
Comment (yhmtsai) [inline .github/format-rebase.sh:33]: $TO_FORMAT is defined in .github/format.sh but it is not sourced

**Comment by yhmtsai** [comment]
Ah... I write the review on the wrong pr.
I will move them to the ginkgo one

**Pull request closed without merging**